envbuilder.go

```diff
@@ -476,7 +480,7 @@ func Run(ctx context.Context, options Options) error {
 			buildParams = &devcontainer.Compiled{
 				DockerfilePath:    dockerfilePath,
 				DockerfileContent: string(content),
-				BuildContext:      options.WorkspaceFolder,
+				BuildContext:      filepath.Join(options.WorkspaceFolder, options.BuildContextPath),
```

Comment:
Since `WorkspaceFolder` is optional, what's the end-result if it's not provided, will this be correct?

Reply:
`WorkspaceFolder` will be given a default value if none is provided: https://github.com/coder/envbuilder/blob/0643d173ef0ad63dae5a7ea3697f16891564b378/envbuilder.go#L270-L276